crates/bevy_winit/src/lib.rs

```diff
@@ -142,12 +144,14 @@ impl<T: Event> Plugin for WinitPlugin<T> {
 #[derive(Debug, Default, Clone, Copy, Event)]
 pub struct WakeUp;
 
-/// The [`winit::event_loop::EventLoopProxy`] with the specific [`winit::event::Event::UserEvent`] used in the [`winit_runner`].
+/// A wrapper type around [`winit::event_loop::EventLoopProxy`] with the specific
+/// [`winit::event::Event::UserEvent`] used in the [`winit_runner`].
 ///
 /// The `EventLoopProxy` can be used to request a redraw from outside bevy.
 ///
 /// Use `NonSend<EventLoopProxy>` to receive this resource.
-pub type EventLoopProxy<T> = winit::event_loop::EventLoopProxy<T>;
+#[derive(Resource, Deref)]
+pub struct EventLoopProxyWrapper<T: 'static>(winit::event_loop::EventLoopProxy<T>);
```

Comment:
The name is a bit long, but wasn't sure what a better name would be. I wanted it to clearly related to the winit type it wraps.

Reply:
I like this name. Totally fine for a very niche type.